docs/components/optimize/userguide/additional-features/filters.md

```diff
@@ -1,38 +1,18 @@
 ---
 id: filters
-title: Filters
+title: Overview
```

Comment:
Note: In general it feels like the filters section actually should be part of Process Analysis as it's present in the edit mode, this would also align with "Decision Analysis" actually having a filter sub-section. I think it originally ended up here as a common filter section but by the time decision and process filtering actually diverged.